samples/led-more-blinking-lights/TimeEnvelope.cs

```diff
@@ -19,10 +19,10 @@ public static void AddTime(IEnumerable<TimeEnvelope> envelopes, int value)
         }
     }
 
-    public TimeEnvelope(int count, bool throwOnOverthrow = true)
+    public TimeEnvelope(int count, bool throwOnOverflow = true)
```

Comment:
I know the class is internal, but shouldn't these methods also be documented, for easier maintenance?

Reply:
I mention this because I noticed that other internal classes (like Bmxx80Register) do have documentation.

Reply:
The bar we have for this package is that you have to document your public API but that's it. Some community members decide to also document their internal APIs but this is optional